README.md

```diff
@@ -308,9 +308,11 @@ Other Style Guides
     // best
     const has = Object.prototype.hasOwnProperty; // cache the lookup once, in module scope.
     console.log(has.call(object, key));
-    /* or */
+    // or
```

Comment:
this style is used intentionally to attract attention; let's revert please.

Reply:
[10.4](https://github.com/airbnb/javascript#modules--no-duplicate-imports)

Do you mind if I change it as shown here then?
Because then we will not have made the mistake mentioned in the document. (18.2)